iree/turbine/kernel/wave/index_sequence_analysis.py

```diff
@@ -62,13 +63,9 @@ def has_strided_access(node: fx.Node) -> bool:
         """
         custom = get_custom(node)
         if isinstance(custom, Write):
-            strides = [
-                simplify_index(custom.register_index[dim]).stride
-                for dim in custom.register_index
-            ]
+            strides = [simplify_index(custom.index[dim]).stride for dim in custom.index]
```

Comment:
Ah, that was leftover from me tinkering, will change back

Reply:
Actually for this, the register_ here is now the ExtractSlice from the gpr_offset chunked write. Seems like these newly generated ExtractSlice do not have an index. Do you think it's better to set indices on these newly generated `ExtractSlices`, or do we keep the current change of doing `custom.index`.

Reply:
I think we should set indices on all ops just because it makes it easier if all ops have indices and then we don't have to think about what if they don't. (I know there are places in the code where we do this index check, but I think we should transition them all to not need the checks)

Reply:
Sounds good! :)

Reply:
Done! :)